### PR TITLE
add a message concerning the robot information

### PR DESCRIPTION
**Pull request by vrabaud**
comments anyone ?

### CODE REVIEW

**Comment by vrabaud** [comment]
the goal is to mirror http://doc.aldebaran.com/2-1/naoqi/motion/tools-general-api.html?highlight=getrobotconfig#ALMotionProxy::getRobotConfig and probably add info over time (like serial number).
This will be published once in a latched manner by naoqi_driver

**Comment by Karsten1987** [comment]
Do we really need all these fields? For example a field has_laser and robot_model for Pepper are redundant, no? Similar issues for number of legs for pepper? According to naoqi there is 1 leg :)

Why is arm_version a bool? Shouldn't it be a uint8 as well for iterating over VERSION_33, VERSION_40 etc ?

**Comment by vrabaud** [comment]
@Karsten1987 , please comment in the file, it makes comments easier to track.
I updated the PR.

**Comment by vrabaud** [comment]
for the laser, NAO can also have a laser on some prototypes.

**Comment by vrabaud** [comment]
renamed a few things and added the service

File: msg/RobotInfo.msg [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+bool has_laser
+# "Extended Arms": True or False.
+bool has_extended_arms
+
+# Number of Legs : 0 or 2
+int32 number_of_legs
+# Number of Arms : 0 or 2
+int32 number_of_arms
+# Number of Hands: 0 or 2
+int32 number_of_hands
```
Comment (Karsten1987) [inline msg/RobotInfo.msg:30]: do we have versions with arms but without hands?

File: msg/RobotInfo.msg [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# "Arm Version"  : "VERSION_32" or "VERSION_33" or "VERSION_40".
+string arm_version
+
+# "Laser"        : True or False.
+bool has_laser
+# "Extended Arms": True or False.
+bool has_extended_arms
+
+# Number of Legs : 0 or 2
+int32 number_of_legs
```
Comment (Karsten1987) [inline msg/RobotInfo.msg:26]: how do we deal with pepper? It has 0 or 1 leg?

File: msg/RobotInfo.msg (hunk 1 above)
Comment (vrabaud) [inline msg/RobotInfo.msg:30]: good point, I guess it was the case at some point ....

**Pull request merged**